nncf/quantization/algorithms/accuracy_control/backend.py

```diff
@@ -0,0 +1,166 @@
+"""
+ Copyright (c) 2023 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+from abc import ABC
+from abc import abstractmethod
+from typing import List, Any
+
+from nncf.common.graph.graph import NNCFGraph
+from nncf.common.graph.graph import NNCFNode
+from nncf.common.graph.operator_metatypes import OperatorMetatype
+
+
+class AccuracyControlAlgoBackend(ABC):
+
+    # Metatypes
+
+    @staticmethod
+    @abstractmethod
+    def get_quantizer_metatypes() -> List[OperatorMetatype]:
+        """
+        Returns a list of quantizer metatypes.
+
+        :return: The list of quantizer metatypes.
+        """
+
+    @staticmethod
+    @abstractmethod
+    def get_const_metatypes() -> List[OperatorMetatype]:
+        """
+        Returns a list of constant metatypes.
+
+        :return: The list of constant metatypes.
+        """
+
+    @staticmethod
+    @abstractmethod
+    def get_quantizable_metatypes() -> List[OperatorMetatype]:
+        """
+        Returns a list of metatypes for operations that may be quantized.
+
+        :return: The list of metatypes for operations that may be quantized.
+        """
+
+    @staticmethod
+    @abstractmethod
+    def get_quantize_agnostic_metatypes() -> List[OperatorMetatype]:
+        """
+        Returns a list of quantize agnostic metatypes.
+
+        :return: The list of quantize agnostic metatypes.
+        """
+
+    @staticmethod
+    @abstractmethod
+    def get_shape_of_metatypes() -> List[OperatorMetatype]:
+        """
+        Returns a list of shape of metatypes.
+
+        :return: The list of shape of metatypes.
+        """
+
+    # Creation of commands
+
+    @staticmethod
+    @abstractmethod
+    def create_command_to_remove_quantizer(quantizer_node: NNCFNode):
```

Comment:
The return type hint was missed here and below.

Reply:
Done